_config.yml

```diff
@@ -14,7 +14,7 @@ linkedin_url: https://www.linkedin.com/company/acm-teach-la-acm-ucla
 editor_link: https://editor.uclaacm.com
 
 general_interest: https://tinyurl.com/teach-la-interest-form
```

Comment:
Shall we leave this in commented? So it’ll be easier to find and switch back to it!!